### PR TITLE
Change commands for new version of poetry

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export PYTHONPATH=$(shell pwd)/redditbot/
 
 dependencies:
+	@pip install -U pip poetry
```
Comment (lucaspolo) [inline Makefile:4]: In the project, I'm assuming that Poetry is already installed in the machine and there are some issues that can be annoying when running Poetry in the same environment that it manages.

Comment (lucaspolo) [inline Makefile:4]: In the project, I'm assuming that Poetry is already installed in the machine and there are some issues that can be annoying when running Poetry in the same environment that it manages.

**Review by lucaspolo — CHANGES_REQUESTED** [review]

**Comment by lucaspolo** [comment]
I forgot to tell you a little thing, the project, because the release-please, is using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), can you change the commit message?

File: Makefile (hunk 1 above)
Comment (jairhenrique) [reply Makefile]: :+1:

**Review by lucaspolo — APPROVED** [review]

**Pull request merged**